### PR TITLE
status verification

### PR DESCRIPTION
**Pull request by Arturoo0**

### CODE REVIEW

File: myftp.py [hunk 1]
```diff
@@ -9,14 +9,19 @@ def initConnection():
   hostName = sys.argv[1]
   if (hostName == 'test'):
     hostName = ftputils.TEST_HOST
-  ftpController = FTPController(hostName)
-  print(ftpController.connect())
-  return ftpController
+    ftpController = FTPController(hostName)
```
Comment (Bruception) [inline myftp.py]: What if the user inputs their own FTP server host? Perhaps we can just use a `sys.exit()` [here](https://github.com/Bruception/CNT4713-Project1/blob/master/myftp.py#L6).

File: myftp.py [hunk 2]
```diff
@@ -9,14 +9,19 @@ def initConnection():
...
+    return ftpController
+  else:
+    print('Invalid host name specified, exiting.')
+    return None
 
 def promptLogin(ftpController):
   username = input('Please enter your username: ')
   password = input('Please enter your password: ')
-  print(ftpController.login(username, password))
+  loginStatusCode = ftpController.login(username, password)
```
Comment (Bruception) [inline myftp.py]: Perhaps we can do this internally within the FTPController's login() method and perform a `sys.exit()`

File: myftp.py [hunk 3]
```diff
@@ -30,8 +35,11 @@ def readCommands(ftpController):
 
 def main():
   ftpController = initConnection()
-  promptLogin(ftpController)
-  readCommands(ftpController)
+  if (ftpController == None):
```
Comment (Bruception) [inline myftp.py]: See comment above.

**Review by Bruception — CHANGES_REQUESTED** [review]
Left some comments.

File: myftp.py (hunk 2 above)
Comment (Arturoo0) [reply myftp.py]: no

File: myftp.py [hunk 4]
```diff
@@ -5,14 +5,15 @@
...
-    return
+    sys.exit()
   hostName = sys.argv[1]
   if (hostName == 'test'):
     hostName = ftputils.TEST_HOST
   ftpController = FTPController(hostName)
   print(ftpController.connect())
   return ftpController
 
+
```
Comment (Bruception) [inline myftp.py]: nit: Extra space

**Review by Bruception — CHANGES_REQUESTED** [review]
LGTM just one nit

**Review by Bruception — APPROVED** [review]
LGTM :)

**Pull request merged**